### DISCUSSION TITLE
NextJS bandwidth usage on Vercel looks inflated

### DISCUSSION THREAD

**Discussion by kristjanmar** [vercel/next.js#27128] [Help] [answered]
Has anyone else noticed that the bandwidth reported in Vercel's usage tab is much higher than the bandwidth actually transferred to the client's web browser?

When I test the site myself with Chrome developer tools open, then view a page or two on my Vercel hosted site, the bandwidth number in Vercel's "Usage" tab goes up *much* more than the actual kB transferred according to Chrome developer tools.

In extreme cases, I might visit a page that takes up 130kB of Bandwidth, but at the same time Vercel's usage tab goes up by 2MB. 

I've tested this multiple times and it always happens. I currently host my development site on an actual domain on Vercel, and there is no one visiting it except me, so I know this can not be explained by someone else visiting the site at the same time.

**Comment by leerob** [comment, marked as answer]
https://github.com/vercel/vercel/discussions/6459#discussioncomment-1696358
